### PR TITLE
Add more test to the test suite 

### LINKED ISSUE #180: As a dev, I'd like to refactor tests to encapsulate the new user repository's methods.
**Acceptance criteria**
- [ ] Assert that **GetUserById** can retrieve an existing user-object via their id.
- [ ] Assert that **GetUserByName** can retrieve an existing user-object via their name.
- [ ] Assert that **GetUserByName** can retrieve an existing user-object , who's got an email, via their email.
- [ ] Assert that **InsertUser** can insert and save a new user-object in the DbSetUser. Check that they exist. 
- [ ] Assert that **DeleteUser** can delete an existing user-object . Check that they do not exist.
- [ ] Assert that **CreateUser** can create and insert a new user-object in the DbSetUser. Check that the user exists.
- [ ] Assert that **FollowUser** inserts a user's id as FollowerId and another user's id as FolowingId.

### PR DESCRIPTION
**Pull request by August-Brandt**
Adds more database tests for methods in the user and author repos.
Resolves #180

### CODE REVIEW

File: src/Chirp.Infrastructure/ChirpRepository/AuthorRepository.cs
Comment (Haruyzal) [inline src/Chirp.Infrastructure/ChirpRepository/AuthorRepository.cs:1]: Is line 117: `int cheepsCount = 0` in use? I can't seem to find it being used.

File: src/Chirp.Infrastructure/ChirpRepository/CheepRepository.cs
Comment (Haruyzal) [inline src/Chirp.Infrastructure/ChirpRepository/CheepRepository.cs:1]: It seems this branch' codebase is outdated, as it keeps pinging missing `await` operations in .e.g CheepRepository, but I'm pretty sure Jonas and Alex had already fixed this...

**Review by Haruyzal — COMMENTED** [review]
The tests themselves look good, I'm just afraid the codebase is fairly outdated 'cause of our unchronological merges...

**Comment by Slug-Boi** [comment]
I would agree with @Haruyzal on this one. I would probably do a merge with main and change the parts that give errors after that. And if anything gives errors then make a new branch and try to fix those from main after the merge

**Comment by August-Brandt** [comment]
This branch has now been update to catch up with main

**Review by JonasSkjodt — APPROVED** [review]
Looking good after you caught up to main! Nice clear test method names as well. Approved

**Review by Slug-Boi — APPROVED** [review]
Looks like you did what we recommended with merging main into the branch. Good job looks ready for merge with main now!

**Review by Alex-itu — APPROVED** [review]

**Pull request merged**